### PR TITLE
Adding pics to BootMii page

### PR DESCRIPTION
**Pull request by JasperZebra**
### Notes

Added pics of BootMii on the BootMii page for better visuals.

### CODE REVIEW

File: _pages/en_US/bootmii.md [hunk 1]
```diff
@@ -37,16 +39,20 @@ If you have BootMii installed as boot2 you will need to launch BootMii by restar
...
+- After this step, it will verify the backup. While it is recommended, it can be skipped by pressing the EJECT button on your Wii. Note that if you have a disc inserted in the disc drive, pressing EJECT will also eject the disc. ![BootMii_Return_Arrow](/images/BootMii/BootMii_NAND_Backup.png)
 5. When the backup is fully complete, exit the NAND backup screen by pressing any button.
-6. To exit BootMii, press the Back button (the one with the arrow) and then you can press either the Wii Menu button or the Homebrew Channel button to exit where you want to.
+
+6. To exit BootMii, press the Back button (the one with the arrow) and then you can press either the Wii Menu button or the Homebrew Channel button to exit where you want to. ![BootMii_Return_Arrow](/images/BootMii/BootMii_Return_Arrow.png)
 
 To restore from a NAND backup on your SD card, you can follow these instructions using RestoreMii (the icon with the red arrow, aka the second icon on your left). This is useful in the unlikely case you brick your Wii.
 {: .notice--info}
+![BootMii_Return_Arrow](/images/BootMii/BootMii_Red_Arrow.png)
```
Comment (TheShadowEevee) [inline _pages/en_US/bootmii.md:55]: This image feels a bit out of place between the two notice blocks, that might just be me though.
image

**Review by TheShadowEevee — CHANGES_REQUESTED** [review]
Other than the one comment, the capitalization on the folder the images are in needs to be fixed (It's BootMIi, not BootMii). That's keeping any of the new images from loading properly

File: _pages/en_US/bootmii.md (hunk 1 above)
Comment (Snare-Hawk) [reply _pages/en_US/bootmii.md:55]: i agree, so i'm going to add it inside the notice box

Comment (JasperZebra) [reply _pages/en_US/bootmii.md]: i just fixed it, how does it look?

**Comment by Snare-Hawk** [comment]
this should not have taken me nearly as long as it did to do what i did
i blame bundle

**Pull request merged**